ci-operator/config/openshift/cluster-kube-scheduler-operator/openshift-cluster-kube-scheduler-operator-master.yaml

```diff
@@ -40,3 +40,11 @@ tests:
   commands: TEST_SUITE=openshift/smoke-4 run-tests run-tests
   openshift_installer:
     cluster_profile: aws
+- as: e2e
```

Comment:
please rename to `e2e-aws-operator`.  `e2e` is too generic.

Reply:
sure